docs/documentation.md

```diff
@@ -0,0 +1,21 @@
+# Introduction to documentations 
```

Comment:
This is a good idea, but I would not have a separate page, I would fold this into the "getting_started" page.

Reply:
always get nervous changing stuff by others >.< heh